Riot/Modules/Room/RoomViewController.m

```diff
@@ -1194,6 +1194,12 @@ + (Class) mainToolbarClass
 - (void)updateRoomInputToolbarViewClassIfNeeded
 {
     Class roomInputToolbarViewClass = [RoomViewController mainToolbarClass];
+
+    // If RTE is enabled, delay the toolbar setup until `completionSuggestionCoordinator` is ready.
+    if (roomInputToolbarViewClass == WysiwygInputToolbarView.class && _completionSuggestionCoordinator == nil)
```

Comment:
Since the hard dependency is the bridge, rather than the UI which is setup in that function, I actually added an extra toolbar update where the bridge is actually created (`displayRoom`).

Reply:
Makes sense to me 👌